Packages/vcs/vcs/Plegend.py

```diff
@@ -115,6 +120,8 @@ def __init__(self, member):
         self.y1 = 0.129999995232
         self.x2 = 0.949999988079
         self.y2 = 0.159999996424
+        self.offset = .015
```

Comment:
@doutriaux1 Would it be possible to have a smaller value here so that most baselines do not change. It seems that for the first images at least, the change is the bigger offset?